examples/multi_package/BUILD

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//python:packaging.bzl", "py_wheel", "py_package")
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # Apache 2.0
+
+py_package(
+    name = "multi_package_example_pkg",
+    packages = [
+        "examples.multi_package.bar",
```

Comment:
from this example I see that py_package takes a packages attribute where I put different values from the ones I put in the py_wheel#packages - they seem to mean different things?

Can we make them the same here? If not, should we give a different name to one of the attributes?

Also it makes me think the docstrings for these attributes need to be more clear what we mean by a "package" since a user might mistakenly give one of these where we expect the other